polling_stations/apps/data_collection/management/commands/misc_fixes.py

```diff
@@ -170,6 +170,66 @@ def handle(self, *args, **kwargs):
             "E06000059", "30801", Point(-1.889868, 50.828982, srid=4326)
         )
 
+        # user issue report #55
+        print("updating: Dorset Fire & Rescue, Peverell Avenue West...")
+        update_station_point(
+            "E06000059", "30244", Point(-2.471750, 50.713013, srid=4326)
+        )
+
+        # user issue report #56
+        print("updating: Higher Failsworth Primary School...")
+        update_station_point(
+            "E08000004", "5063", Point(-2.148726, 53.514296, srid=4326)
+        )
+
+        # user issue report #65
+        print("updating: Penistone Cricket Club...")
+        update_station_point("E08000016", "110", Point(-1.618343, 53.525626, srid=4326))
+
+        # user issue report #66
+        print("updating: St Mary's Catholic Primary School...")
+        update_station_point(
+            "E06000040",
+            "23-st-marys-catholic-primary-school",
+            Point(-0.724786, 51.535229, srid=4326),
+        )
+        update_station_point(
+            "E06000040",
+            "24-st-marys-catholic-primary-school",
```

Comment:
nice. I totally forgot to mention the "same station address with 2 different ids" edge case yesterday. Well remembered :D